packages/channels/botpress-channel-messenger/src/outgoing.js

```diff
@@ -13,55 +13,37 @@ const handlePromise = (next, promise) => {
 const handleText = (event, next, messenger) => {
   return handlePromise(
     next,
-    messenger.sendTextMessage(event.raw.to, event.raw.message, event.raw.quick_replies, event.raw)
+    messenger.sendTextMessage(event.raw.to, event.raw.message, event.raw.quick_replies, event.raw, event.page.id)
```

Comment:
It's not good add fifth params. It'll be better use something like this;
```javascript
messenger.sendTextMessage(event.raw.to, event.raw.message, event.raw.quick_replies, Object.assign({}, event.raw, { pageId: event.page.id})

Reply:
Does 4 or 5 params really make a difference?

Adding pageId as the final param just makes this API call consistent with all others...

Reply:
Yes, it's. Because in fututre maybe we want add one more and after that we will have 6 params. It can be repeat again after that 7 params and etc.